src/components/common/AlertModal/index.tsx

```diff
@@ -0,0 +1,34 @@
+import type { ReactElement } from 'react'
+import { Styled } from './styled'
+import type { AlertModalProps } from './types'
+
+const AlertModal = ({
+  isOpen = true,
+  isCheckShape = false,
```

Comment:
```suggestion
  hasCheckIcon = false,
```
체크 아이콘이 보여지는 형태여서 has로 표현되면 좋을 것 같은데 위 변수명은 어떠신가요?

Reply:
완료!
[링크](https://github.com/price-offer/offer-fe/pull/108/commits/2f6a4f180e5cc19c7c09ed82b4805a718940c178#diff-08768223cb51d41ee8726f9e11b78640fecff8209b3c03c0986f90775f3fb7f8)